Block/AccountBlockService.php

```diff
@@ -25,7 +25,7 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class AccountBlockService extends BaseBlockService
+class AccountBlockService extends AbstractBlockService
```

Comment:
Just figured out that there is already an interface for this `AdminBlockServiceInterface`